Source/OxyPlot/Rendering/OxyCircle.cs

```diff
@@ -0,0 +1,197 @@
+        }
+
+        /// <summary>
+        /// Returns a <see cref="System.String" /> that represents this instance.
+        /// </summary>
+        /// <returns>A <see cref="System.String" /> that represents this instance.</returns>
+        public override string ToString()
+        {
+            return string.Format(CultureInfo.InvariantCulture, "({0}, {1}, {2})", this.Center.X, this.Center.Y, this.Radius);
+        }
+
+        /// <summary>
+        /// Returns a <see cref="System.String" /> that represents this instance.
+        /// </summary>
+        /// <param name="format">The format.</param>
+        /// <param name="formatProvider">The format provider.</param>
+        /// <returns>
+        /// A <see cref="System.String" /> that represents this instance.
+        /// </returns>
+        public string ToString(string format, IFormatProvider formatProvider)
+        {
+            const string Separator = ", ";
+            var builder = new StringBuilder();
+            builder.Append("(");
+            builder.Append(this.Center.X.ToString(format, formatProvider));
+            builder.Append(Separator);
+            builder.Append(this.Center.Y.ToString(format, formatProvider));
+            builder.Append(Separator);
+            builder.Append(this.Radius.ToString(format, formatProvider));
+            builder.Append(")");
+            return builder.ToString();
+        }
+
+        /// <summary>
+        /// Returns a circle that is expanded or shrunk by the specified radius amounts.
+        /// </summary>
+        /// <param name="dr">The amount by which to expand or shrink the radius of the circle.</param>
+        /// <returns>The expanded/shrunk <see cref="OxyCircle" />.</returns>
+        public OxyCircle Inflate(double dr)
+        {
+            return new OxyCircle(this.Center, this.Radius + dr);
+        }
+
+        /// <summary>
+        /// Returns a circle that is moved by the specified horizontal and vertical amounts.
+        /// </summary>
+        /// <param name="offsetX">The amount to move the circle horizontally.</param>
+        /// <param name="offsetY">The amount to move the circle vertically.</param>
+        /// <returns>The moved <see cref="OxyCircle" />.</returns>
+        public OxyCircle Offset(double offsetX, double offsetY)
+        {
+            return new OxyCircle(this.Center.X + offsetX, this.Center.Y + offsetY, this.Radius);
+        }
+
+        /// <summary>
+        /// Intersects this <see cref="OxyCircle"/> with another <see cref="OxyCircle"/> with an equal center point.
+        /// </summary>
+        /// <param name="circle">The other <see cref="OxyCircle"/>.</param>
+        /// <returns>The intersection between this <see cref="OxyCircle"/> and the other <see cref="OxyCircle"/>.</returns>
+        /// <exception cref="System.ArgumentException">circle;The center should be equal to this center.</exception>
+        public OxyAnnulus Intersect(OxyCircle circle)
```

Comment:
I think `Intersect` is the wrong word here, though I'm not entirely sure what the idea of this operation is

Reply:
I used the class OxyRect as template for OxyCircle and adapted the functionality to "circles"... Might be deleted.

Reply:
I think keep the APIs simple for the time being: it's easy to add more methods when we need them.